Berlin/PetermannIINachtrag57.xml

```diff
@@ -29,29 +29,243 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0333SBB"/>
+                  <collection>Nachtrag Petermann</collection>
                   <idno facs="https://content.staatsbibliothek-berlin.de/dc/799375586/manifest">Petermann II Nachtrag 57</idno>
-                 
                   <altIdentifier>
-                     <idno>799375586</idno>
+                     <idno>Dillmann 62</idno>
                   </altIdentifier>
                   <altIdentifier>
-                     <idno>Dillmann 62</idno>
+                     <idno>799375586</idno>
                   </altIdentifier>
                </msIdentifier>
-               <msContents/>
+               <msContents>
+                  <msItem xml:id="ms_i1">
+                     <locus from="1ra" to="72vb"/>
+                     <title ref="LIT2598Yosipp" type="complete"/>
+                     <msItem xml:id="ms_i1.1">
+                        <locus from="1ra1" to="1ra11"/>
+                        <title ref="LIT2598Yosipp#Introduction" type="incomplete"/>
+                        <note>Introduction with many variant readings.</note>
+                     </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="1ra11" to="4rb18"/>
+                        <title ref="LIT2598Yosipp#Kefl1" type="complete"/>
+                        <msItem xml:id="ms_i1.2.1">
+                           <locus from="1ra11" to="1va18"/>
+                           <title ref="LIT2598Yosipp#Section1"/>
+                           <incipit xml:lang="gez">አዳም፡ አውለደ፡ ሴትሃ።</incipit>
```

Comment:
it seems a bit too short to help with text identification. If possible, it would be nice to have a longer portion